roguelike-core/src/main/java/greymerk/roguelike/treasure/loot/provider/ItemNovelty.java

```diff
@@ -162,16 +162,18 @@ public static RldItemStack greymerksHatchet() {
         .withEnchantment(Enchantment.Effect.KNOCKBACK)
         .withEnchantment(Enchantment.Effect.UNBREAKING.atLevel(2))
         .asStack()
-        .withDisplayName("Greymerk's Hatchet")
-        .withDisplayLore(TextFormat.DARKGREEN.apply("Pointlessly sharp"));
+        .withDisplayName("novelty.roguelike.greymerk.name")
+        //.withDisplayLore(TextFormat.DARKGREEN.apply("Pointlessly sharp"));
```

Comment:
I notice that you move the colour formatting away from `TextFormat.DARKGREEN ...` and into colour codes inside of the translated text in the files en_us.lang and en_mx.lang.

I'm fine with it, and I'd like to learn and follow any new conventions you're creating. To that end I need to understand why having the control codes inside of the translated text is preferable to appending the colour codes to the text post-translation. Maybe the reason is because it would have been a lot of work to refactor. Let me know! 😄

Reply:
Hi!
Well there are a couple of reasons.
1. When I was translating the commands, if the result message had color, on chat I had a weird character (probably the § inside the code) at the beginning of the message.
2. I've seen other mods like Dynamic Trees, when they hardcode the text color using the character §  instead of using TextFormatting inside the code (for example, The Woodland Staff description), the weird character shows at the beginning in the tooltip.

So, since you're not using TextFormatting but your own enum structure, I didn't want the weird character to pop up inside the tooltip. 

So I thought, maybe using § inside the lang file, you won't have to deal with Forge-specific stuff (because it's a vanilla thing afaik). Besides, I think in future versions you can also control the text format inside the json file.